src/librustc_middle/ty/mod.rs

```diff
@@ -228,7 +228,7 @@ impl AssocItem {
     pub fn def_kind(&self) -> DefKind {
```

Comment:
This should probably be on `AssocKind` (and renamed to `as_def_kind`).

Reply:
Moved.